src/reference-designs/tko-on-aws.md

```diff
@@ -25,7 +25,8 @@ The following network diagram shows the network layout used with this reference
 
 This reference design uses Tanzu Kubernetes Grid to manage the lifecycle of multiple Kubernetes workload clusters by bootstrapping a Kubernetes management cluster with the Tanzu command line tool. Consider the following when configuring the network for Tanzu Kubernetes Grid:
 
-* Use an internal load balancer scheme. We recommend creating an internal load balancer as a best practice to avoid exposing the Kubernetes API to the public Internet. To use an internal load balancer, customize the Tanzu Kubernetes Grid templates with a `kustomize` override. If you use an internal load balancer, run Tanzu Kubernetes Grid from a machine with access to the target VPC private IP space.
+* Use an internal load balancer scheme. We recommend creating an internal load balancer as a best practice to avoid exposing the Kubernetes API to the public Internet.To avoid creating a public-facing load balancer you can set AWS_LOAD_BALANCER_SCHEME_INTERNAL to true in the cluster configuration file `AWS_LOAD_BALANCER_SCHEME_INTERNAL: true`
```

Comment:
```suggestion
* Use an internal load balancer scheme. A best practice is to create an internal load balancer to avoid exposing the Kubernetes API to the public Internet. To avoid creating a public-facing load balancer, you can set AWS_LOAD_BALANCER_SCHEME_INTERNAL to true in the cluster configuration file `AWS_LOAD_BALANCER_SCHEME_INTERNAL: true`
```